code/game/machinery/autolathe.dm

```diff
@@ -77,6 +79,12 @@
 	materials.retrieve_all()
 	return ..()
 
+/obj/machinery/autolathe/proc/OnToolAttack(datum/source, atom/tool, mob/user)
```

Comment:
```suggestion
/obj/machinery/autolathe/proc/on_tool_attack(datum/source, atom/tool, mob/user)
```
Procs use snake_case, I'm pretty sure.

Reply:
Materials container uses camel case for the function used in the signaling stuff but this may be a one time or really old thing so ill do this